### DISCUSSION TITLE
stat-computing-r-python/part-wrangling/03-data-cleaning

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#12] [General]
# stat-computing-r-python/part-wrangling/03-data-cleaning

https://srvanderplas.github.io/stat-computing-r-python/part-wrangling/03-data-cleaning.html

**Comment by allycjay** [comment]
The code for mutating in 19.6 does not work.

**Reply by srvanderplas** [reply]
Fixed in 981867d8bc8157f4533791f734de2d70669b4f9d

**Comment by srvanderplas** [comment]
storm_strength_duration = storms.groupby(["name", "year"]).agg(duration = ("time", lambda x: max(x) - min(x)),max_strength = ("category", "max"))

ggplot(aes(x = "factor(max_strength)", y = "duration"), data = storm_strength_duration) + geom_boxplot()
## TypeError: ggplot.__init__() got multiple values for argument 'data'

plotnine needs named arguments for aesthetics -- fix

**Comment by henryrscharf** [comment]
Just bringing this to the attention of the author: Some interesting error printouts in the `dplyr` example in 22.5 stemming from "`! could not find function "str_extract"`". Works on my machine though.
